aana/tests/units/test_image_chat_dialog.py

```diff
@@ -0,0 +1,183 @@
+# ruff: noqa: S101
+
+from importlib import resources
+
+import pytest
+
+from aana.core.models.image import Image
+from aana.core.models.image_chat import (
+    ImageChatDialog,
+    ImageChatMessage,
+    ImageContent,
+    TextContent,
+)
+
+
+@pytest.fixture
+def input_image_path():
+    """Gets an Image for test."""
+    image_path = resources.path("aana.tests.files.images", "Starry_Night.jpeg")
```

Comment:
Why don't you create a fixture for the Image object? `Image(path=input_image_path)`
BTW, you don't need to set `save_on_disk=False` if you provide path, it's already saved on disk.

Reply:
I can do it.
There is a problem when the image is saving on the disk, or sending it over the remove function, which load the numpy image and there will be the typeerror for jsonifying it for test_cache.

Reply:
I tried to use the image_path, so when creating multiple images with the input, it does not point to the same object

Reply:
You don't need to use test cache for idefics deployment. Test cache is about to be removed anyway.